### ISSUE TITLE
revert change of license ?

### ISSUE THREAD

**Issue by VolkerH** [nvladimus/npy2bdv#5]
Hi, 
I just noticed that you changed the license from BSD-3/MIT to GPL some time back.
I bundled/vendored an earlier version of this code here in a project for easier distribution (while it was still under BSD-license).

**Comment by nvladimus** [comment]
Hi, @VolkerH,
Thanks for your question. In my understanding, there should be no problem with the new GPL license as long as you keep the source code open. The GPL many restricts closing the source when you distribute the code/product. We can discuss it in detail, please DM me. Thanks!

**Comment by VolkerH** [comment]
Ok, as long as you don't see a problem with my use in [here](https://github.com/VolkerH/Ultramicroscope2BigStitcher).
I originally included your code for ease of building, but I saw yesterday that one can meanwhile just install `npy2bdv` from `pypi` so I now changed my code so that `npy2bdv` is a dependency rather than including a copy in my repo. 
Thanks again for this project.
Will close this issue.
